credentials_template

```diff
@@ -0,0 +1,7 @@
+ORG =  #: URL to AGOL organization (https://www.arcgis.com)
```

Comment:
what is this file? should it have an extension?

Reply:
Good catch, thanks.